src/test/java/org/broadinstitute/hellbender/tools/spark/utils/LargeLongHopscotchSetTest.java

```diff
@@ -155,7 +155,7 @@ void loadRandomLongsTest() {
         for (int valNo = 0; valNo != HHASH_NVALS; ++valNo) {
             final long randLong = randomLong(rng);
             hopscotchSet.add(randLong);
-            hashSet.add(new Long(randLong));
+            hashSet.add(Long.valueOf(randLong));
```

Comment:
we can probably just rely on autoboxing in most of these places

Reply:
I've fixed these in #6145.